src/overlays/actors/ovl_En_Horse_Game_Check/z_en_horse_game_check.c

```diff
@@ -138,8 +138,8 @@ s32 func_808F8CCC(EnHorseGameCheck* this, PlayState* play2) {
 
     this->unk_17C = -1;
     Camera_ChangeSetting(play->cameraPtrs[CAM_ID_MAIN], CAM_SET_FREE0);
-    Play_CameraSetAtEye(play, 0, &sp40, &sp4C);
-    Play_CameraSetFov(play, 0, 45.0f);
+    Play_SetCameraAtEye(play, 0, &sp40, &sp4C);
```

Comment:
Can name these.

Reply:
Same as above, named in a following PR

Reply:
(I'll resolve all the comments that are already being prepped in a follow-up PR)